test/test_format.py

```diff
@@ -419,7 +505,7 @@ def test_nice_year(self):
                 self.assertTrue(len(nice_year(dt, lang=lang)) > 0)
                 # Looking through the date sequence can be helpful
 
-#                print(nice_year(dt, lang=lang))
+    #                print(nice_year(dt, lang=lang))
```

Comment:
stray `print()` (not part of this PR, but pylint seems to have exposed it. thanks, pylint!)